### PR TITLE
build/bump-dependencies

### PR DESCRIPTION
**Pull request by Axelen123**

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/backdrops/misc/pro/ProUnlockPatch.kt [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 import app.revanced.extensions.exception
-import app.revanced.patcher.annotation.Description
-import app.revanced.patcher.annotation.Name
 import app.revanced.patcher.data.BytecodeContext
 import app.revanced.patcher.extensions.InstructionExtensions.addInstruction
 import app.revanced.patcher.extensions.InstructionExtensions.getInstruction
 import app.revanced.patcher.patch.BytecodePatch
-import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patcher.patch.annotation.Patch
 import app.revanced.patches.backdrops.misc.pro.annotations.ProUnlockCompatibility
```
Comment (validcube) [inline src/main/kotlin/app/revanced/patches/backdrops/misc/pro/ProUnlockPatch.kt:9]: Not needed

File: src/main/kotlin/app/revanced/patches/backdrops/misc/pro/ProUnlockPatch.kt [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 import com.android.tools.smali.dexlib2.iface.instruction.OneRegisterInstruction
 
-@Patch
-@Name("Pro unlock")
-@Description("Unlocks pro-only functions.")
+@Patch(
+    name = "Pro unlock",
+    description = "Unlocks pro-only functions."
+)
 @ProUnlockCompatibility
```
Comment (validcube) [inline src/main/kotlin/app/revanced/patches/backdrops/misc/pro/ProUnlockPatch.kt:17]: Use new format

```kt
@Patch(
  compatiblePackages = [
    CompatiblePackage(
      "<package name>",
      arrayOf[<versions>]
     )
  ]
)
```

**Review by validcube — CHANGES_REQUESTED** [review]
Blargh

**Comment by oSumAtrIX** [comment]
Please fix merge conflicts after moving patches to the correct directory. Examples:

/patch/some.kt -> /some.kt
/bytecode/fingerprints/ -> /fingerprints
/resource/some.kt -> /some.kt
...

**Comment by oSumAtrIX** [comment]
Superseded by #2946

**Pull request closed without merging**